Runtime/Scripts/RGBotLocalRuntime/SampleBot/RGSampleBot.cs

```diff
@@ -20,7 +21,7 @@ protected override RGBotLifecycle GetBotLifecycle()
 
         public override void ConfigureBot(RG rgObject)
         {
-            var classIndex = new System.Random().Next(4);
+            var classIndex = new Random().Next(4);
             var classes = new string[] {"Mage", "Rogue", "Tank", "Archer"};
```

Comment:
I know this was in a previous PR, but note that sometime in the next week or two, we may want to provide a default implementation that works out of the box for any game, like just printing the state that was received.